tests/DataSourcePouchDBTests.js

```diff
@@ -168,10 +168,36 @@ fluid.defaults("kettle.tests.dataSource.15.CouchDB.URL.set.existing", {
     }
 });
 
+fluid.defaults("kettle.tests.dataSource.16.CouchDB.URL.set.specialChars", {
+    gradeNames: "kettle.tests.dataSource.pouchDB.write.environment",
+    name: "14. Testing CouchDB URL datasource with HTTP - set with data containing special characters",
+    dataSourceMethod: "set",
+    dataSourceModel: {
+        test: "Gerät"
+    },
+    components: {
+        dataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                gradeNames: "kettle.dataSource.CouchDB",
```

Comment:
Given this is a fix to general URL handling, this test should simply be against the URL dataSource and not allocated to the CouchDB space